src/programs/sum_all_eer_files/sum_all_eer_files.cpp

```diff
@@ -0,0 +1,215 @@
+#include "../../core/core_headers.h"
+#include <wx/dir.h>
+
+class
+        SumAllEer : public MyApp {
```

Comment:
For programs in cisTEM, I think the convention is to match the file name and the app name, so this should be SumAllEerFiles. (It also is good to keep parity with other similar programs sum_all_tif(mrc)_files)

Reply:
This is in parity with the other programs, the other app names do not include files